studio/pages/sign-in-mfa.tsx

```diff
@@ -31,8 +32,9 @@ const SignInMfaPage: NextPageWithLayout = () => {
           const { data, error } = await auth.mfa.getAuthenticatorAssuranceLevel()
           if (error) {
             // if there was a problem signing in via the url, don't redirect
+            toast.error(`Failed to retrieve assurance level: ${error.message}`)
```

Comment:
the only reason it will fail to retrieve the assurance level is if the session is missing (https://github.com/supabase/gotrue-js/blob/45b6e3e656f842c60ebe2709f59bad4eb6f5811e/src/GoTrueClient.ts#L2292) - we should just return an error saying that the user is not logged in and ask them to retry